pytorch_toolkit/ote/ote/tests/test_case.py

```diff
@@ -165,18 +180,23 @@ def setUpClass(cls):
 
             download_snapshot_if_not_yet(cls.template_file, cls.template_folder)
 
+        def setUp(self):
+            self.output_folder = os.path.join(self.template_folder, f'output_{self.id()}')
```

Comment:
please make appropriate changes in training_extensions/pytorch_toolkit/object_detection/tests/common/test_case.py on lines: 53, 59

Reply:
Oh! I see! thank you very much!

Reply:
Fixed